.github/workflows/pr.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: SwiftLint
         uses: docker://norionomura/swiftlint:0.54.0_swift-5.9.0
         with:
-          args: swiftlint --reporter github-actions-logging --strict
+          args: swiftlint --reporter github-actions-logging --strict --quiet
```

Comment:
I had a couple SwiftLint warnings in this PR, but the logs for SwiftLint on CI are noisy because they print every single file being evaluated, even if they have no violation.

`--quiet` makes it so that only violations are logged, it will now be easier to open a failed SwiftLint run and see what went wrong.